src/fetchers/git-fetcher.js

```diff
@@ -15,22 +16,61 @@ const fs = require('fs');
 const invariant = require('invariant');
 
 export default class GitFetcher extends BaseFetcher {
-  _fetch(): Promise<FetchedOverride> {
-    const {protocol, pathname} = url.parse(this.reference);
-    if (protocol === null && typeof pathname === 'string') {
-      return this.fetchFromLocal(pathname);
-    } else {
-      return this.fetchFromExternal();
+  async getLocalAvailabilityStatus(): Promise<bool> {
+    const tarballLegacyMirrorPath = this.getTarballMirrorPath({withCommit: false});
```

Comment:
I think this justifies a comment explaining what is legacyMirrorPath and why we support it